DNN Platform/Dnn.AuthServices.Jwt/Package/01.00.00.SqlDataProvider

```diff
@@ -0,0 +1,172 @@
+/************************************************************/
```

Comment:
is this file supposed to be in here twice?  looks like the `Package` directory may have been checked in accidentally